sync.php

```diff
@@ -42,6 +51,7 @@
     }
```

Comment:
If the `or true` is correct, you should just execute `sync_students()` and lines 45-47

Reply:
damn only checked for || true that was a mistake, removed.